thamos/cli.py

```diff
@@ -1466,7 +1466,8 @@ def images(
                 )
                 key = Text.from_markup(key, style="green bold")
                 # Here, key value are ignored. This is a trick to have "table header" as the first column.
-                table_content.extend([{"key": key, "value": value}])
+                if value:
```

Comment:
It may be clearer and more efficient to add something like:
```python
if not value:
    continue
```
at the beginning of the `for` loop instead of waiting for here at the end

Reply:
Fixed, ready for review :+1: